frontend/src/stores/ParticipantStore.js

```diff
@@ -44,15 +44,15 @@ export class ParticipantStore {
   }
   setVisit = data => {
     this.visit = data
+    // swapping extraneous program ans service objects for numbered IDs, required for visit request
```

Comment:
If the code is not useful anymore I vote to delete rather than comment.

Reply:
yup this can and will be deleted in another push.